src/Migrations/M036MigrateCoversContentBlockToPostsListBlock.php

```diff
@@ -0,0 +1,360 @@
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'post_tag', 'separator' => ' ']
+                                        ),
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'category', 'separator' => ' | ']
+                                        ),
+                                    ],
+                                    ['className' => 'posts-list-meta']
+                                ),
+                                Utils\Functions::create_new_block(
+                                    Utils\Constants::BLOCK_TITLE
+                                ),
+                                Utils\Functions::create_new_block(
+                                    Utils\Constants::BLOCK_EXCERPT
+                                ),
+                                Utils\Functions::create_group_block(
+                                    [
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_AUTHOR,
+                                            ['isLink' => true]
+                                        ),
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_DATE
+                                        ),
+                                    ],
+                                    ['className' => 'posts-list-meta',]
+                                ),
+                            ],
+                            []
+                        ),
+                    ]
+                ),
+            ],
+            [
+                'lock' => [
+                    'move' => true,
+                    'remove' => true,
+                ],
+            ]
+        );
+    }
+
+    /**
+     * Create and get a new navigation link block.
+     *
+     * @return array - The new block.
+     */
+    private static function get_nav_links_block(): array
+    {
+        return Utils\Functions::create_new_block(
+            Utils\Constants::BLOCK_NAV_LINK,
+            [
+                'label' => 'See all stories',
+                'url' => get_permalink(get_option('page_for_posts')),
+                'className' => 'see-all-link',
+            ]
+        );
+    }
+
+    /**
+     * Create and get a new query-no-results block.
+     *
+     * @return array - The new block.
+     */
+    private static function get_query_no_results_block(): array
+    {
+        return Utils\Functions::create_block_query_no_results(
+            [
+                Utils\Functions::create_block_paragraph(
+                    [],
+                    'No posts found. (This default text can be edited)'
```

Comment:
This should be translatable. But also, does it mean this is what will be displayed in the frontend if not changed? Shouldn't it just be used as a placeholder in the editor?